sample_app/sampleapp_common.c

```diff
@@ -196,6 +196,117 @@ void app_print_network_details (
    printf ("Current Gateway:      %s\n", gateway_string);
 }
 
+static app_subslot_t * app_subslot_get(
```

Comment:
All functions, especially in the sample app, need docstrings.

Reply:
Done